### ISSUE TITLE
Integrate and test WebMarks 

### ISSUE THREAD

**Issue by mr-mig** [WorldBrain/infrastructure#7]

**Comment by lwm** [comment]
Is this a duplcate of #9 ?

**Comment by mr-mig** [comment]
No, this is basically about making all services work together with correct ports and stuff. 
Mailer, queue, db, bla bla bla. 
As you have noticed there is a hustle around docker containers in Hypothesis :)
